### DISCUSSION TITLE
(suggestion) Icon instead of Number

### DISCUSSION THREAD

**Discussion by ArthurGamerHD** [auto94/HyperX-Cloud-2-Battery-Monitor#4] [Ideas]
Since i’m not really an fan of having small texts on my taskbar, i have a suggestion that is having an option to display as an icon on the taskbar,

i’m not an artist myself but i work with User Interfaces in C# (not really experienced C++), 
i did an mockup (functional) in my fork of this code for an example of a battery icon that reacts with the percentage (and uses the same colors as current background settings) using basic pens and rectangles, looks like that: 

![IMG_1036](https://github.com/auto94/HyperX-Cloud-2-Battery-Monitor/assets/11463584/5432c8a4-8684-4bd3-ba0b-3b9d89c45df9)

i personally think looks way better, and specially with the Core Alpha, that have over 300 hours of battery time, is not really necessary have an accurate image every time (and also, the tooltip shows the exactly number)

**Comment by auto94** [comment]
Yeah, that looks good and I can see the exact number being pointless if the battery lasts 300hrs. 

Since I've personally grown used to the number and I think for a 30hrs battery it is kind of useful, I'd make the old way (number) stay as an optional setting maybe?

**Comment by auto94** [comment]
This was added a couple of releases ago. Closing discussion.
